packages/forms-web-app/__tests__/unit/utils/query-string-builder.test.js

```diff
@@ -3,64 +3,82 @@ const { queryStringBuilder } = require('../../../src/utils/query-string-builder'
 describe('utils/query-string-builder', () => {
 	const valueToExpect = {
 		input1: {
-			params: null,
+			queryParams: null,
 			includedParams: []
 		},
 		input2: {
-			params: {},
-			includedParams: null
+			queryParams: {},
+			includedParams: null,
+			append: true
 		},
 		input3: {
-			params: {
+			queryParams: {
```

Comment:
The query param is what goes after the ? in the url. The param is what goes after the first slash. In this case this attribute should have a name of params in my opinion.

Reply:
Adding a console.log to application/forms in the controller documents.js file we can see the difference the queryParams normally have: page, type, category, stage attributes. The params have everything the query params have plus the info inside the slashes: caseRef, classification, page, type, category, stage.